pkg/sidecar/param/pub_api.go

```diff
@@ -22,16 +22,16 @@ type Cerializer interface {
 }
 
 type fuseParamsBundleParams struct {
-	Name         string `json:"name" yaml:"name"`
-	SrcPath      string `json:"srcPath" yaml:"srcPath"`
-	SrcRepo      string `json:"srcRepo" yaml:"srcRepo"`
-	SrcLabel     string `json:"srcLabel" yaml:"srcLabel"`
-	SrcBundle    string `json:"srcBundle" yaml:"srcBundle"`
-	DestPath     string `json:"destPath" yaml:"destPath"`
-	DestRepo     string `json:"destRepo" yaml:"destRepo"`
-	DestMessage  string `json:"destMessage" yaml:"destMessage"`
-	DestLabel    string `json:"destLabel" yaml:"destLabel"`
-	DestBundleID string `json:"destBundleID" yaml:"destBundleID"`
+	name         string `json:"name" yaml:"name"`
```

Comment:
@ransomw1c as per our discussion, changing attributes to unexported in effect disables unmarshalling

Reply:
further, moving attributes to unexported brings no particular values here, since the container type is already unexported.